src/lib.rs

```diff
@@ -0,0 +1,23 @@
+pub mod rng;
+
+/// Simple example of a function that returns a string
+/// 
+/// # Example
+/// 
+/// ```
+/// use libelinvar::hello;
+/// hello();
+/// ```
+pub fn hello() -> String {
+    return "Elinvar".to_owned();
+}
+
+#[cfg(test)]
```

Comment:
Long term, should this be pulled into a separate "test" module? Or is it generally more common for tests to be near the methods it tests?